content/docs/rfcs/10/README.md

```diff
@@ -138,7 +143,7 @@ In addition to `/vac/waku/*` protocols, Waku v2 MAY directly use the following l
 /ipfs/ping/1.0.0
 ```
 
-for liveness checks between peers, or to keep peer-to-peer connections alive.
+For liveness checks between peers, or to keep peer-to-peer connections alive.
```

Comment:
This line is part of the sentence

Waku v2 MAY directly use the following protocols:
.... for liveness checks,
... with protocol IDs .... as basic means for capability discovery

so should not be capitalised or end with a full stop.